duckduckgo/iOS
# Fix missing image reference that was breaking Today Widget prototype

**Description**:
Fixes a broken image and updates the fonts of the Today widget prototype. This is SUPER low priority! I wanted a screenshot of the prototype for an O-A proposal and needed to fix this issue.

**Steps to test this PR**:
<img width="400" alt="addingtodayprototype" src="https://user-images.githubusercontent.com/1652407/40441095-e4578214-5eb7-11e8-9a30-ac3416ff622c.png">

1. Add the Today extension to the project. From the project file, go to Targets, select DuckDuckGo, General, press the "+" button under Embedded Binaries, select "TodayExtension.appex"
1. Run the app and add some bookmarks
1. Go to the Today screen and add the DuckDuckGo widget
1. Ensure it renders

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
